api/src/controllers/controllerUtil.ts

```diff
@@ -1,14 +1,14 @@
 import { NextFunction, Request, Response } from 'express';
 
-type Handler = (req: Request, res: Response, next: NextFunction) => Promise<void>;
+type Handler<P, RS, RQ, Q, L> = (req: Request<P, RS, RQ, Q, L>, res: Response<RS, L>, next: NextFunction) => Promise<void>;
```

Comment:
Had to do some fancy generic stuff b/c I was having trouble with typing. Happy to go into details if people are interested.

Reply:
These are just the parameters listed in [`RequestHandler`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/df6c94b0ebb1446f1be2ccde75d8abb2340e205c/types/express/index.d.ts#L119-L124) right?

Reply:
That is correct

Reply:
Sounds good 😄